### PR TITLE
add autoplayout selector

### PR DESCRIPTION
**Pull request by michael-grace**
button mc button face....also why can't i merge it...please be an easy fix...

### CODE REVIEW

**Comment by michael-grace** [comment]
wait...its not conflicting

**Comment by michael-grace** [comment]
wait...it is....

File: src/showplanner/AutoPlayoutModal.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  close: () => any;
+}
+
+export function AutoPlayoutModal(props: AutoPlayoutProps) {
+  return (
+    <Modal isOpen={props.isOpen} onRequestClose={props.close}>
+      <div>
+        <h1 className="d-inline">
+          <FaPlayCircle className="mx-2" size={30} />
+          URY Automatic Playout
```
Comment (markspolakovs) [inline src/showplanner/AutoPlayoutModal.tsx:17]: #branding

File: src/showplanner/AutoPlayoutModal.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          <FaTimes />
+        </Button>
+      </div>
+      <hr />
+      <iframe
+        id="uploadIframe"
+        src={process.env.REACT_APP_MYRADIO_NONAPI_BASE + "/NIPSWeb/playout/"}
+        frameBorder="0"
+        title="URY Automatic Playout"
+      ></iframe>
```
Comment (markspolakovs) [inline src/showplanner/AutoPlayoutModal.tsx:33]: Screm

**Review by markspolakovs — APPROVED** [review]

**Pull request merged**